index.html

```diff
@@ -4419,57 +4294,31 @@ <h3>Parsing content</h3>
     </section>
   </section>
 
-  <section class="informative"> <h2>Security mechanisms</h2>
+  <section class="informative"> <h2>Security mechanisms for implementations</h2>
     <p>
       In order to support security policies concerning Web NFC, the following
-      mechanisms are recommended for the implementations.
+      mechanisms are recommended for implementations.
```

Comment:
There's only one mechanism.

Reply:
Right. Actually I removed those introductory sentences, they don't add any value, just make it longer.